webpack/assets/javascripts/react_app/components/CommonParameterForm/CommonParamFormHelpers.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Button, Icon } from 'patternfly-react';
+
+import StringValue from './components/values/StringValue';
+import NumberValue from './components/values/NumberValue';
+import BooleanValue from './components/values/BooleanValue';
+import Editor from './components/values/Editor';
+
+export const fieldSizeByType = type => {
+  switch (type) {
+    case 'integer':
+    case 'real':
+      return 'col-md-4';
+    case 'boolean':
```

Comment:
Can bool be `col-md-4` as well?
![Screenshot from 2020-12-09 14-44-11](https://user-images.githubusercontent.com/30431079/101631455-0367b680-3a2d-11eb-8c47-477470f7b126.png)